src/SDKs/SqlManagement/Sql.Tests/SessionRecords/Sql.Tests.BlobAuditingTest/TestServerBlobAuditing.json

```diff
@@ -825,7 +825,7 @@
         ],
         "User-Agent": [
           "FxVersion/4.6.25009.01",
-          "Microsoft.Azure.Management.Resources.ResourceManagementClient/1.0.0-preview"
+          "Microsoft.Azure.Management.ResourceManager.ResourceManagementClient/1.0.0-preview"
```

Comment:
@nathannfan how is this possible, if you are using 1.6.0 version of resource manager with the new namespace, how come you are still recording the old version?